lib/node/http.js

```diff
@@ -165,6 +166,37 @@ class HTTP extends Server {
       });
     });
 
+    // Balance by address
+    this.get('/balance/:address', async (req, res) => {
+      const valid = Validator.fromRequest(req);
+      const address = valid.str('address');
```

Comment:
You might also want to accept a `minconf` parameter. Either that, or include "unconfirmedBalance" kind of stuff in the output. Unconfirmed coins will have a height of `-1` I think

Reply:
We're on the same page. This morning I started updating this PR to resemble the wallet balance API, i.e. `unconfirmed`, `confirmed`, `lockedUnconfirmed`, etc.